### DISCUSSION TITLE
BYD Seal U does not work with Profile for BYD Atto 3

### DISCUSSION THREAD

**Discussion by cbeham1** [meatpiHQ/wican-fw#283] [General]
Hello,
got my Seal U yesterday. But i cant see anything except Status of meatpi (online/offline) if i choose Atto 3 profile!
can anyone help me please!
thx

![image](https://github.com/user-attachments/assets/f358ddf8-be6a-4bfb-86c9-17c835c7383e)

**Comment by meatpiHQ** [comment]
Have you tried using car scanner app? does anything show up?

According to ABRP app the ATTO and the Seal U share the same PIDs.

**Reply by cbeham1** [reply]
sorry, cant get any connection any more (ODB adapter)
there is a AP WiCAN_xxxx and i can connect with standardpassword @meatpi#, but with adress 192.168.80.1 there is no response...
plugout and in again was no success :(

what can i do?

**Reply by meatpiHQ** [reply]
What is the status of the LEDs? Is the Blue LED ON? Did you have sleep mode enabled?

**Reply by cbeham1** [reply]
Hello and good evening,
there is one blue led if i plug in CANBUS (power). the other 2 (WLAN) and 3rd (?) is off.
